plugins/org.jboss.tools.cdi.reddeer/src/org/jboss/tools/cdi/reddeer/validators/BeansXmlUIValidationProvider.java

```diff
@@ -13,12 +13,13 @@
 import org.eclipse.reddeer.eclipse.ui.views.markers.ProblemsView.ProblemType;
 import org.jboss.tools.cdi.reddeer.annotation.ValidationType;
 
-public class BeansXmlUIValidationProviderCDI11 extends AbstractValidationProvider {
+public class BeansXmlUIValidationProvider extends AbstractValidationProvider {
 	
-	private final String jsr = "JSR-346";
-	
-	public BeansXmlUIValidationProviderCDI11() {
+	private static String jsr;
```

Comment:
I would suggest not to use static here. you are assigning it in constructor.

Reply:
Fixed.